docs/success-guide/setup_config_files.md

```diff
@@ -126,25 +126,44 @@ Limits on deletion prevent accidental account deletion in the event of misconfig
 
-	user_removal:
-	    - exclude_group: special_users   
-	    - exclude:   ".*@example.com"   
-	    - exclude_adobe_id              
+	dashboard_user_exclusions:
+        exclude_groups: 
+            - administrators   # Names an Adobe user group or product configuration whose members are not to be altered or removed by User Sync
+            - contractors      # You can have more than one group in a list
+        exclude_users:
+            - ".*@example.com"
+            - important_user@gmail.com
+        exclude_identity_types:
+            - adobeID          # adobeID, enterpriseID, and/or federatedID
+                          
+
+In the above, administrators, contractors, and the user names are example values.  You would use the names of Adobe user groups, product configurations, or users that you have created.
+
+`exclude_groups` defines a list of Adobe user groups or product configurations. Adobe users who are members of such groups are neither removed, updated, nor have their group membership changed.
+
+`exclude_users` gives a list of patterns. Adobe users with usernames that match (default case-insensitive, unless the pattern specifies case-sensitive) any of the specified patterns (if there are more than one) are neither removed, updated, nor have their group membership changed.
+
+`exclude_identity_types` gives a list of identity types. Adobe users who have one of these identity types are neither removed, updated, nor have their group membership changed.
+
+Note that:
+
+- You generally wouldn't use all three exclude options.
+
+- Directory side users are still created on the Adobe side even if one of the exclude parameters would then exclude the user on the Adobe side from updates in successive runs. That is, these parameters apply only to Adobe users who exist when the Adobe directory is being matched against the customer directory.
+
+- The entire `dashboard_user_exclusions` configuration section is optional. 
+
+- Accounts that would have been removed or updated but were not because of this feature are listed as `info` level log entries.
+
+- Note that Federated accounts that are not in the directory cannot log in anyway (because login is handled by the ID provider and the user is no longer listed there) even if the account still exists in the Adobe dashboard.
```

Comment:
- take out "Note that".  This is a bullet inside of a "note that" list of items :).
- "not in the directory" -> "not in the customer directory" (or "disabled in the customer directory" is even better)